server/db_setup/create_schema.sql

```diff
@@ -17,4 +17,20 @@ CREATE TABLE FileTypeMap (
     file_type varchar(255),    
     upload_date DATETIME,
     is_uploaded boolean DEFAULT 0
+);
+
+DROP TABLE IF EXISTS User_File_SocketMap;
+
+CREATE TABLE User_File_SocketMap (    
+    user_name varchar(255),
+    file_unique_id varchar(36) NOT NULL,
+    current_socket_id varchar(255),
+    current_state varchar(255) 
+);
```

Comment:
The `User_File_SocketMap` table should have a primary key to ensure each record is unique. Consider adding a primary key constraint on `file_unique_id` or a composite key that includes both `user_name` and `file_unique_id`. This will help maintain data integrity.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*